### PR TITLE
ZFIN-8345

### PR DESCRIPTION
**Pull request by cmpich**

### CODE REVIEW

File: source/org/zfin/feature/repository/HibernateFeatureRepository.java [hunk 1]
```diff
@@ -501,41 +507,28 @@ public String getPrefixById(int labPrefixID) {
...
-            }
-        });
+        String hqlLab = """ 
+            select distinct lfp.organization from OrganizationFeaturePrefix lfp
+            where lfp.featurePrefix is not null
+            and lfp.organization.name is not null
+            and lfp.currentDesignation = :true
+            """;
+        Query<Organization> query = HibernateUtil.currentSession().createQuery(hqlLab, Organization.class);
+        query.setParameter("true", true);
```
Comment (rtaylorzfin) [inline source/org/zfin/feature/repository/HibernateFeatureRepository.java:529]: This is a minor thing, but I usually remove parameters like this and just update the hql to be "currentDesignation = true"

File: source/org/zfin/feature/repository/HibernateFeatureRepository.java [hunk 2]
```diff
@@ -438,40 +442,42 @@ public List<FeatureLabEntry> getFeaturesForPrefix(String prefix) {
     @Override
     public List<OrganizationFeaturePrefix> getOrganizationFeaturePrefixForPrefix(String prefix) {
         String hql = " select distinct lfp from OrganizationFeaturePrefix lfp , Feature f, FeaturePrefix fp "
-                + " where f.featurePrefix = fp "
-                + " and fp.prefixString = :featurePrefix "
-                + " and lfp.featurePrefix = fp "
-                + "";
-        return currentSession().createQuery(hql).setParameter("featurePrefix", prefix).list();
+                     + " where f.featurePrefix = fp "
```
Comment (rtaylorzfin) [inline source/org/zfin/feature/repository/HibernateFeatureRepository.java:445]: maybe a text block here too

**Review by rtaylorzfin — APPROVED** [review]
This looks good to me! If you wanted, you could always add more text blocks.  I usually convert `setBoolean("varname", true)` to just an inline boolean expression in the where clause.

File: source/org/zfin/feature/repository/HibernateFeatureRepository.java (hunk 1 above)
Comment (cmpich) [reply source/org/zfin/feature/repository/HibernateFeatureRepository.java]: Sure.

**Pull request merged**